packages/foreman/rubygem-actionmailbox/rubygem-actionmailbox.spec

```diff
@@ -16,16 +17,16 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 # start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby >= 2.5.0
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(activesupport) = 6.0.2.2
-Requires: %{?scl_prefix}rubygem(activerecord) = 6.0.2.2
-Requires: %{?scl_prefix}rubygem(activestorage) = 6.0.2.2
-Requires: %{?scl_prefix}rubygem(activejob) = 6.0.2.2
-Requires: %{?scl_prefix}rubygem(actionpack) = 6.0.2.2
+Requires: %{?scl_prefix_ruby}ruby(rubygems) 
+Requires: %{?scl_prefix}rubygem(activesupport) = 6.0.3.1
+Requires: %{?scl_prefix}rubygem(activerecord) = 6.0.3.1
+Requires: %{?scl_prefix}rubygem(activestorage) = 6.0.3.1
+Requires: %{?scl_prefix}rubygem(activejob) = 6.0.3.1
+Requires: %{?scl_prefix}rubygem(actionpack) = 6.0.3.1
 Requires: %{?scl_prefix}rubygem(mail) >= 2.7.1
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}ruby >= 2.5.0
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel 
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
```

Comment:
I wonder why there is no obsoletes here, is mailbox new in 6.0?

Reply:
Aye